### DISCUSSION TITLE
Deployment/build notification

### DISCUSSION THREAD

**Discussion by igor-topolski** [coollabsio/coolify#1431] [Feature Requests]
It would be amazing if there was an option to configure e.g. email, telegram or Slack notification upon failed/successful deployment.

Current UI sometimes makes it hard when you navigate between multiple projects to see how your deployments are doing

**Comment by andrasbacsai** [comment]
You can have Email / Telegram / Discord notifications. 

Maybe it is not so visible on the UI, but if you go to `/team/notifications`, you can see them - UI needs to be redesigned a bit. 😅
